### PR TITLE
Add mdrenderer

### PR DESCRIPTION
**Pull request by perexg**
According https://github.com/lepture/mistune/issues/56 , trying to put this extension to mistune-contrib.

### CODE REVIEW

File: mistune_contrib/mdrenderer.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      text, type, t = MdRenderer.get_block(text)
+      if type == 'l':
+        r += (ordered and ('# ' + t) or ('* ' + t)) + '\n'
+    return r
+
+  def list_item(self, text):
+    return 'l' + str(len(text)) + ':' + text
+
+  def block_code(self, code, lang=None):
+    return '```no-highlight\n' + code + '\n```\n'
```
Comment (lepture) [inline mistune_contrib/mdrenderer.py]: why `no-highlight` ?

File: mistune_contrib/mdrenderer.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+This class renders parsed markdown back to markdown.
+It is useful for automatic modifications of the md contents.
+
+## Authors and License
+
+Copyright (C) 2015 Jaroslav Kysela
+
+License: WTFPL 2
+"""
```
Comment (lepture) [inline mistune_contrib/mdrenderer.py]: doc string should be in rst.

**Comment by unode** [comment]
Is there any chance this might get merged?
At the moment it seems like none of the python markdown parsers out there can output markdown.

It's a blatantly obvious missing feature despite being requested on almost all of them.

In other words, there's not a single markdown library for python that can parse markdown into some internal representation and re-create markdown from this representation.

**Comment by lepture** [comment]
ping @perexg

you need to fix things I mentioned.

**Comment by perexg** [comment]
@lepture : updated, let me know, if you want more changes .

File: mistune_contrib/mdrenderer.py (hunk 2 above)
Comment (lepture) [reply mistune_contrib/mdrenderer.py]: ping @perexg 

docstring should be in RST, not Markdown.

File: mistune_contrib/mdrenderer.py [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+-------------------
+
+Copyright (C) 2015 Jaroslav Kysela
+
+License: WTFPL 2
+"""
+
+from mistune import Renderer
+
+class Object:
```
Comment (lepture) [inline mistune_contrib/mdrenderer.py]: remove this useless class.

**Review by lepture — CHANGES_REQUESTED** [review]

**Comment by perexg** [comment]
Removed Object class, but I don't see the doc string issue problem. Tried two online RST editors and the result seems fine.

**Comment by lepture** [comment]
@perexg e.g. https://github.com/lepture/mistune-contrib/blob/master/mistune_contrib/highlight.py

Make your doc string like this.

**Comment by perexg** [comment]
Done.

**Pull request merged**